Gruntfile.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = function (grunt) {
+
+    // Load external grunt task config.
+    grunt.loadTasks('./grunt');
+
+    grunt.loadNpmTasks('grunt-contrib-jshint');
+    grunt.loadNpmTasks('grunt-jscs');
```

Comment:
We normally use `jit-grunt`, see https://github.com/incuna/teva-becon/blob/6ca1856ad934b5bbe3cd9a9b5eb708bff5b32a1d/Gruntfile.js#L5-L19